CRM/Core/BAO/Address.php

```diff
@@ -1349,7 +1349,7 @@ public static function buildOptions($fieldName, $context = NULL, $props = array(
       case 'world_region':
       case 'worldregion':
       case 'worldregion_id':
-        return CRM_Core_PseudoConstant::worldRegion();
+        return CRM_Core_BAO_Country::buildOptions('region_id', $context);
```

Comment:
Nice and this will eventually call
CRM_Core_PseudoConstant::get('CRM_Core_BAO_Country', $fieldName, 'region_id',[], $context);

@eileenmcnaughton can you pass the $props parameter too? i.e.
```php
        return CRM_Core_BAO_Country::buildOptions('region_id', $context, $props);
```
just in case :)

Reply:
done

Reply:
Thanks